### DISCUSSION TITLE
zh/best-practices/run-downloaded-binaries-on-nixos

### DISCUSSION THREAD

**Discussion by giscus** [ryan4yin/nixos-and-flakes-book#165] [Announcements]
# zh/best-practices/run-downloaded-binaries-on-nixos

一份非官方的新手指南

https://nixos-and-flakes.thiscute.world/zh/best-practices/run-downloaded-binaries-on-nixos

**Comment by ponymushama** [comment]
为了配置 `mason.nvim` 用了 `nix-ld` ，还是很方便的，一次性解决所有 non-nix 二进制文件执行问题。现在 `nix-ld` 有很好用的默认配置，直接开启 `programs.nix-ld.libraries` 就好了。
